charts/sourcegraph/templates/prometheus/prometheus.ServiceAccount.yaml

```diff
@@ -1,12 +1,11 @@
 {{- if and .Values.prometheus.enabled .Values.prometheus.serviceAccount.create -}}
 apiVersion: v1
-imagePullSecrets:
-- name: docker-registry
```

Comment:
shall we clean this up here as well?

https://github.com/sourcegraph/deploy-sourcegraph/blob/master/base/prometheus/prometheus.ServiceAccount.yaml#L2-L3

Reply:
https://github.com/sourcegraph/deploy-sourcegraph/pull/4152